xwayland/xwm.c

```diff
@@ -365,6 +368,36 @@ static void read_surface_class(struct wlr_xwm *xwm,
 	wlr_signal_emit_safe(&surface->events.set_class, surface);
 }
 
+#ifdef WLR_HAS_XCB_ICCCM
```

Comment:
Why does this require ICCCM?

Reply:
Thanks for the explanation. I've removed the `#ifdef`.